### PR TITLE
helm: Fix and add missing podLabels

### PR DESCRIPTION
**Pull request by Subreptivus**
- Pod Labels defined in `values` file were not propagated to actual manifests
- Some of the Pods were missing `podLabels` definition in `values` file and in actual manifests (i.e. certgen jobs Pods)

### CODE REVIEW

**Comment by nathanjsweet** [comment]
@Subreptivus can you run the command `make -C install/kubernetes docs` and commit the changes. We'll have test failures until you update the `quick-install.yaml` file. Thanks!

File: install/kubernetes/cilium/values.yaml [hunk 1]
```diff
@@ -612,6 +614,10 @@ hubble:
     #
     podAnnotations: {}
 
+    # -- Labels to be added to hubble-relay pods
+    #
```
Comment (qmonnet) [inline install/kubernetes/cilium/values.yaml]: Nit: We can probably remove this empty comment line

File: install/kubernetes/cilium/values.yaml [hunk 2]
```diff
@@ -726,6 +732,10 @@ hubble:
     #
     podAnnotations: {}
 
+    # -- Labels to be added to hubble-ui pods
+    #
```
Comment (qmonnet) [inline install/kubernetes/cilium/values.yaml]: And here.

**Review by qmonnet — CHANGES_REQUESTED** [review]
Thank you! Can we get rid of the third commit (the merge, I suspect it was created because you tried to update the branch automatically in GitHub)? No need to update unless there are conflicts to solve, in which case it looks cleaner to rebase the PR on top on the current master.

**Comment by Subreptivus** [comment]
@qmonnet Done, now it should be allright.
As for empty lines, I was just used few lines up to be consistent with a config that was there before (sometimes there were empty lines, sometimes not). So I've removed all empty lines. Hope you're happy now.

**Review by nbusseneau — APPROVED** [review]

**Review by qmonnet — APPROVED** [review]
I'm happy now :). Thank you

**Comment by qmonnet** [comment]
test-me-please

**Review by nebril — APPROVED** [review]

**Review by errordeveloper — APPROVED** [review]

**Pull request merged**